examples/combined_ODNP.py

```diff
@@ -218,13 +214,9 @@
 vd_data.set_prop("postproc_type", IR_postproc)
 nodename = vd_data.name()
 # {{{ again, implement a file fallback
-with h5py.File(
-    os.path.normpath(os.path.join(target_directory, f"{filename}"))
-) as fp:
+with h5py.File(os.path.normpath(os.path.join(target_directory, f"{filename}"))) as fp:
```

Comment:
very soon, please make a new PR to implement your new function.  Note that you will also want to pass it the log info as a kwarg.

Reply:
AG showed me an example so that I can do this

Reply:
there was something added to the file, which I deleted.  that would have been doubled up (if it worked) and we also want to do this in a separate PR.

Reply:
@wfkincaid please don't resolve conversations -- I use resolving them as a way to know which issues are taken care of, and if I can trust others haven't resolved them, it moves much faster